test/clojure-mode-font-lock-test.el

```diff
@@ -161,7 +161,7 @@ POS."
     (should (eq (clojure-test-face-at 2 3) 'font-lock-type-face))
     (should (eq (clojure-test-face-at 5 14) 'font-lock-type-face))))
 
-(ert-deftest clojure-mode-syntax-table/namespace ()
+(ert-deftest clojure-mode-syntax-table/segmented-symbol ()
```

Comment:
It seems to me that those tests should actually fail now - from what I gather by your changes we aim to font-lock as ns-es only ns symbols that are in the `ns` macro.

Reply:
> Perhaps this is the better approach indeed. 

Glad to hear it :)

> It seems to me that those tests should actually fail now - from what I gather by your changes we aim to font-lock as ns-es only ns symbols that are in the ns macro.

Yea, I'll work on it. (I just wanted to hear your general opinion in the first place.) I'll tell you when have something to show.